frontend/src/lib/range-plugin.ts

```diff
@@ -0,0 +1,207 @@
+/* eslint-disable */
+// @ts-nocheck
```

Comment:
Was this supposed to be a temporary thing to let you get it working?

Reply:
this is a copy of the plugin.  I discovered a bug in it while testing this and made a PR, but it's not reviewed/approved/merged yet.  Rather than wait, I just included the source here.  flatpickr project had different linting and code rules so some things complained, rather than fix them up to our rules, I just turned it off